.appveyor.yml

```diff
@@ -38,7 +40,7 @@ after_test:
   - cmd: wkhtmltopdf -V
   - cmd: gwtc -v --css wiki/export.css -t "Teamscale Ephemeral .NET Profiler" -f pdf wiki
   # Create release zip with profiler & documentation
-  - cmd: 7z a %release_zip% .\Profiler\bin\Release\*.dll documentation.pdf
+  - cmd: 7z a %release_zip% .\Profiler\bin\Release\*.dll documentation.pdf .\Uploader\bin\Release\Uploader.* .\Uploader\bin\Release\*.dll .\Uploader\bin\Release\*.xml .\Uploader\bin\Release\*.json .\Uploader\bin\Release\*.config -xr!*.pdb
```

Comment:
I think it would be good to have the uploader in a subdirectory. just from looking at the release zip, it contains quite some files already toplevel :/